src/Workspaces/Core/Portable/Shared/Utilities/AsyncBatchingWorkQueue`0.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Shared.TestHooks;
+
+namespace Roslyn.Utilities
+{
+    /// <inheritdoc cref="AsyncBatchingWorkQueue{TItem, TResult}"/>
+    internal class AsyncBatchingWorkQueue : AsyncBatchingWorkQueue<bool>
+    {
+        public AsyncBatchingWorkQueue(
+            TimeSpan delay,
+            Func<CancellationToken, Task> processBatchAsync,
+            CancellationToken cancellationToken)
+            : this(delay,
+                   processBatchAsync,
+                   asyncListener: null,
+                   cancellationToken)
+        {
+        }
```

Comment:
❓ This constructor is problematic. Was it needed for something?

Reply:
let me check and see if i can remove

Reply:
removed.  though i need eyes on the fallout.